terraform/shared/database/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "aws_region" {
+  type        = string
+  description = "region output by cloud foundry service-key command"
+  default     = "us-gov-west-1"
+}
+
+variable "cf_api_url" {
+  type        = string
+  description = "cloud.gov api url"
+  default     = "https://api.fr.cloud.gov"
+}
+
+variable "cf_org_name" {
+  type        = string
+  description = "cloud.gov organization name"
+  default     = "hhs-acf-ohs-hses"
+}
+
+variable "cf_password" {
+  type        = string
+  description = "secret; cloud.gov deployer account password"
+  sensitive   = true
+}
+
+variable "cf_space_name" {
```

Comment:
[issue] This variable (or at least it's default value) should be in the `stage` directory and not the shared one.